BUILDING.md

```diff
@@ -130,6 +130,12 @@ Depending on the host platform, the selection of toolchains may vary.
 #### Windows
 
 * Visual Studio 2017 with the Windows 10 SDK on a 64-bit host.
+    * Targeting Windows on ARM (ARM64 Win32) requires
```

Comment:
IIUC this is for cross-compilation. Do we know what is needed for building on native Windows/ARM64?

Reply:
Yes, this is for cross-compilation. Visual Studio 2017 only has x64- and x86-hosted toolchains, though they support ARM32, ARM64, x86, and x64. Fortunately, Windows 10 on ARM comes with an x86 emulation system, so the x86 VS toolchain runs as-is. TL;DR: the x86 toolchain cross-compiles for ARM64 on ARM64 just fine, and it's installed the same way as on x64. :)

For Node.js, however, there is a snag. The v8 part of the build has a few tools like mksnapshot that are built then run. Since GYP currently [only supports one target architecture at a time using the MSVS generator](https://github.com/refack/GYP/blob/master/pylib/gyp/generator/msvs.py#L1849), cross-compiling from x64 results in ARM64 tools that cannot run. (At least mksnapshot can be worked around with the 'vcbuild nosnapshot' argument.) Adding that capability requires a lot of work that I'm not actually chartered to do. :(

Building on an ARM64 device solves this problem, since the x86 tools running emulated produce ARM64 tools that can run, but without ARM64 CI/CD this isn't a very practical approach if you don't have a device. (Well...unless you're [ambitious](https://thedigitallifestyle.com/w/index.php/2019/01/31/getting-windows-10-on-arm-up-and-running-on-a-lumia-950xl/))